yt/yt/server/lib/exec_node/config.cpp

```diff
@@ -564,6 +564,9 @@ void TGpuManagerDynamicConfig::Register(TRegistrar registrar)
 
     registrar.Parameter("gpu_info_source", &TThis::GpuInfoSource)
         .Default();
+
+    registrar.Parameter("default_nvidia_driver_capabilities", &TThis::DefaultNvidiaDriverCapabilities)
+        .Default();
```

Comment:
Here we should decide on our default

https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/docker-specialized.html#driver-capabilities

It can be "" (now). This way nvidia's default will be applied

It can be "utility,compute". This way we will fix current nvidia's default to avoid unexpected changes

It can be "all". This way we will avoid manual actions in case we will need to enable some capabilities, by paying some undefined price. [This old question](https://github.com/NVIDIA/nvidia-container-runtime/issues/112) says:
> For most practical purposes, it should be fine

Reply:
I'd start with a nvidia default